Formula/glibc.rb

```diff
@@ -64,9 +108,11 @@ class Glibc < Formula
     sha256 "654794e9e18c2401f1101a3fcf0a85eda448b4b969e9a99782a3f4f4659feda4" => :x86_64_linux
   end
 
-  depends_on "binutils" => :build # binutils 2.20 or later is required
+  depends_on "binutils" => :build # binutils 2.22 or later is required
```

Comment:
```suggestion
  depends_on "binutils" => :build
```
This comment was stale, and will likely become stale again. Better to remove it.